code_schemes/s01e01.json

```diff
@@ -0,0 +1,361 @@
+{
+  "SchemeID": "Scheme-d4290fc2",
+  "Name": "S01E01",
+  "Version": "0.0.0.1",
+  "Codes": [
+    {
+      "CodeID": "code-E-720cdb66",
+      "CodeType": "Meta",
+      "MetaCode": "escalate",
+      "DisplayText": "meta: ESCALATE",
+      "NumericValue": -100080,
+      "StringValue": "escalate",
+      "VisibleInCoda": true,
+      "Shortcut": "e"
+    },
+    {
+      "CodeID": "code-Q-a5d3700d",
+      "CodeType": "Meta",
+      "MetaCode": "question",
+      "DisplayText": "meta: question",
+      "NumericValue": -100010,
+      "StringValue": "question",
+      "VisibleInCoda": true,
+      "Shortcut": "q"
+    },
+    {
+      "CodeID": "code-S-461bfcdf",
+      "CodeType": "Meta",
+      "MetaCode": "statement",
+      "DisplayText": "meta: statement",
+      "NumericValue": -100090,
+      "StringValue": "statement",
+      "VisibleInCoda": true,
+      "Shortcut": "s"
+    },
+    {
+      "CodeID": "code-5850d9b7",
+      "CodeType": "Normal",
+      "DisplayText": "about coronavirus",
```

Comment:
Please confirm these are the L3 codes for this project, as there are many more here than usual